klt-cert-manager/controllers/keptnwebhookcontroller/config.go

```diff
@@ -11,7 +11,8 @@ const (
 	DeploymentName             = "klc-controller-manager"
 	ServiceName                = "klc-webhook-service"
 	SuccessDuration            = 3 * time.Hour
-	Webhookconfig              = "klc-mutating-webhook-configuration"
+	MutatingWebhookconfig      = "klc-mutating-webhook-configuration"
```

Comment:
we should eventually rename these to `klt`

Reply:
I thought we agreed the controller should be called klc?